### ISSUE TITLE
vite-plugin-iso-import is broken with recent versions of SvelteKit / Vite / vite-plugin-svelte

### ISSUE THREAD

**Issue by kevinrenskers** [bluwy/vite-plugin-iso-import#2]
It seems that Vite 2.7.x is breaking vite-plugin-iso-import. No error is given, but the imports simply don't work. This started to happen since SvelteKit 202, which updated to Vite 2.7.

I've now pinned the following package versions, and then everything still works:

```
"@sveltejs/kit": "1.0.0-next.201",
"@sveltejs/vite-plugin-svelte": "1.0.0-next.30",
"vite": "2.6.14",
```

**Comment by bluwy** [comment]
Thanks for bringing this up. Vite 2.7 had a breaking change to the plugin hooks that handle SSR. We had handled that for vite-plugin-svelte but I forgot about this package as well 😅. This should be fixed in v0.1.3.

**Comment by kevinrenskers** [comment]
Thanks!
